job_definitions/code_backups.yml

```diff
@@ -44,6 +44,7 @@
           credentials-id: github_com_and_enterprise
           branches:
             - master
+            - main
```

Comment:
What happens for those repos where the `main` branch has not yet been created?

Reply:
Good question... I'm going to apply the change and find out!

Reply:
I've applied the changes to Jenkins with `make jenkins TAGS=jobs`, and repos which don't have a `main` branch yet work fine still: here's the [git polling log for digitalmarketplace-utils](https://ci.marketplace.team/view/Code%20backups/job/backup-digitalmarketplace-utils/scmPollLog/) as an example.

However, repos which have had their default branch renamed are not so happy :( the [most recent backup of digitalmarketplace-test-utils](https://ci.marketplace.team/view/Code%20backups/job/backup-digitalmarketplace-test-utils/503/console) failed with the error `remote: error: refusing to delete the current default branch 'refs/heads/master'.` 😢

Reply:
Hmm. Do we just need to SSH into Jenkins and clear the cached workspace for this job? I think that might solve the error...

Reply:
I will do some experiments...

Reply:
I think this might be more to do with what's happening on AWS CodeCommit, based on this stack exchange thread

https://superuser.com/questions/748058/how-do-i-delete-origin-master-in-git